### ISSUE TITLE
Disable self tracing

### ISSUE THREAD

**Issue by jlsalmon** [opentracing-contrib/java-specialagent#531]
I am noticing a number of traces from the `okhttp` integration where the `http.url` is the address of my Jaeger collector. These "self traces" don't seem very useful to me. Is it possible to disable these, while retaining other traces gathered from the `okhttp` integration?

I'm using v1.6.1.

**Comment by jlsalmon** [comment]
Note that this behaviour was not present with v1.5.8.

**Comment by safris** [comment]
Hi @jlsalmon, thank you for reporting this issue. The SpecialAgent has specific precautions to prevent from "self tracing". In order to understand what is happening, would you be able to provide a log file with `-Dsa.log.level=FINER`?

What I'd be looking for are entries like this:
```
-------> Intercept [ThreadAgentRule@main]: public synchronized void java.lang.Thread.start()
```

`ThreadAgentRule` is the rule class, `main` is the thread's name, and the rest is the method signature being intercepted.

Can you isolate the exact `okhttp` calls coming from Jaeger that are "self traces"?

**Comment by safris** [comment]
Hi @jlsalmon, I wanted to check in with you regarding this issue. Do you have a log file you can share?

**Comment by safris** [comment]
@jlsalmon, this issue has been resolved and released with SpecialAgent v1.7.3.
